mediator/med_phases_history_mod.F90

```diff
@@ -453,6 +459,10 @@ subroutine med_phases_history_write(gcomp, rc)
           end if
        enddo
 
+       ! Write out areas to history file
+       do n = 1,ncomps
```

Comment:
Is something missing here?

Reply:
Good catch. Forgot to delete that do loop. 
The B1850 test now seems to be running fine.